src/main/kotlin/mate/academy/service/FigureSupplier.kt

```diff
@@ -0,0 +1,75 @@
+package mate.academy.service
+
+import kotlin.math.round
+import kotlin.random.Random
+import mate.academy.model.enums.AvailableFigures
+import mate.academy.model.enums.Color
+import mate.academy.model.figure.Circle
+import mate.academy.model.figure.Figure
+import mate.academy.model.figure.IsoscelesTrapezoid
+import mate.academy.model.figure.Rectangle
+import mate.academy.model.figure.RightTriangle
+import mate.academy.model.figure.Square
+
+const val DEFAULT_RADIUS: Double = 10.0
```

Comment:
In case if these values are not used outside the FigureSupplier class consider making them as private

Reply:
fixed